undefined_syms.txt

```diff
@@ -1,11 +1,4 @@
 // libultra OS symbols
-D_80000000 = 0x80000000;
-D_80000002 = 0x80000002;
-D_80000004 = 0x80000004;
-D_80000008 = 0x80000008;
-D_8000000C = 0x8000000C;
-D_80000010 = 0x80000010;
-D_80000020 = 0x80000020;
 osTvType = 0x80000300;
 osRomBase = 0x80000308;
 osResetType = 0x8000030C;
```

Comment:
Are these going to stay as undefined_syms forever?

Reply:
We have two options. We can use a separate linker file, identical to `undefined_syms`, but containing only these symbols with a more appropriate name. Or, we can create a spec segment like the buffers segment starting at `0x80000300` and include a file with these as bss symbols. Creating a file for them may be slightly misleading however, since they're uneditable.

Reply:
I personally like the idea of a separate linker file for those

Reply:
Since, as far as I understand, these and the A4...s are used to talk to the hardware directly and are uneditable, they probably should be but somewhere that reflects that.

Reply:
The A4...s at least are removable, I was able to successfully get rid of those in the handwritten asm PR

Reply:
Ah, so it would just be these few. Hmm, well same thing applies, although the file'll be shorter. I think an undefined_syms equivalent it the better option, since it looks less editable.